src/subscription.ts

```diff
@@ -528,11 +564,17 @@ export class Subscription extends Subscriber {
    *   const apiResponse = data[0];
    * });
    */
-  getMetadata(gaxOpts?, callback?) {
-    if (is.fn(gaxOpts)) {
-      callback = gaxOpts;
-      gaxOpts = {};
-    }
+  getMetadata(callback?: RequestCallback<Subscription>):
```

Comment:
I think for the overloads we don't want to have callback be optional, either its absent and the method returns a promise or its present and we return void.